app/src/main/java/com/jerboa/Utils.kt

```diff
@@ -474,7 +474,7 @@ fun communityNameShown(community: Community): String =
 
 fun hostName(url: String): String? =
     try {
-        URL(url).host
```

Comment:
Whats this for?

Reply:
Its to show the hostname of the site for any link posts. The only thing changed here is to remove the `www.`, which a lot of links have and wasted screen space.

Reply:
Create a different function for that. And only use for the "Shown" purposes. But right now it is also used functionality wise to compare between hosts.

Reply:
Done